### PR TITLE
Crew Plushies! Redtail, Annie, Paxton, Mason.

### PR DESCRIPTION
**Pull request by Sishen1542**
Added plushies for Paxton, Annie, Chase Redtail & Mason.
Cause we totally need more of these ;)

### CODE REVIEW

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -595,6 +595,21 @@
...
+	icon_state = "annie"
+	item_state = "annie"
+	attack_verb = list("blorbled", "slimed", "absorbed", "glomped")
+
+/obj/item/toy/plush/slimeplushie/paxton
+	name = "slime plushie"
+	desc = "An adorable stuffed toy that resembles a slimey crewmember."
+	icon_state = "paxton"
+	item_state = "paxton"
+	attack_verb = list("blorbled", "slimed", "absorbed", "glomped", "CQC'd", "jabroni'd", "powergamed", "robusted", "cakehatted")
```
Comment (Ghommie) [inline code/game/objects/items/plushes.dm]: Self-insertion detected, what a shame.

File: code/game/objects/items/plushes.dm [hunk 2]
```diff
@@ -595,6 +595,21 @@
...
 
+/obj/item/toy/plush/slimeplushie/annie
+	name = "slime plushie"
+	desc = "An adorable stuffed toy that resembles a slimey crewmember."
+	icon_state = "annie"
+	item_state = "annie"
+	attack_verb = list("blorbled", "slimed", "absorbed", "glomped")
+
+/obj/item/toy/plush/slimeplushie/paxton
+	name = "slime plushie"
```
Comment (Ghommie) [inline code/game/objects/items/plushes.dm]: This value is the same of the parent type, remove.

File: code/game/objects/items/plushes.dm [hunk 3]
```diff
@@ -595,6 +595,21 @@
 	squeak_override = list('sound/effects/blobattack.ogg' = 1)
 	gender = FEMALE	//given all the jokes and drawings, I'm not sure the xenobiologists would make a slimeboy
 
+/obj/item/toy/plush/slimeplushie/annie
+	name = "slime plushie"
```
Comment (Ghommie) [inline code/game/objects/items/plushes.dm]: Ditto.

File: code/game/objects/items/plushes.dm [hunk 4]
```diff
@@ -595,6 +595,21 @@
...
+	desc = "An adorable stuffed toy that resembles a slimey crewmember."
+	icon_state = "annie"
+	item_state = "annie"
+	attack_verb = list("blorbled", "slimed", "absorbed", "glomped")
```
Comment (Ghommie) [inline code/game/objects/items/plushes.dm]: Honestly, you should move the "glomped" attack string to the parent type and remove this line altogether.

File: code/game/objects/items/plushes.dm (hunk 1 above)
Comment (Sishen1542) [reply code/game/objects/items/plushes.dm]: ;)

**Review by deathride58 — APPROVED** [review]

**Pull request merged**